### PR TITLE
support azure ad authentication

### PR DESCRIPTION
**Pull request by sear-lin-avepoint**
we append azure ad authentication to our connector

### CODE REVIEW

**Comment by sear-lin-avepoint** [comment]
I saw there is an error in the properties validation, but it does work,  and is guided by MS connector team.

File: certified-connectors/AvePoint Cloud Governance/apiProperties.json [hunk 1]
```diff
@@ -1,6 +1,36 @@
...
   "properties": {
     "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "uiDefinition": {
+          "displayName": "Username"
+        },
+        "oAuthSettings": {
+          "identityProvider": "aad",
+          "clientId": "1892da54-7f57-4b6a-b6b0-fdc7f44e04c9",
```
Comment (Amjed-Ayoub) [inline certified-connectors/AvePoint Cloud Governance/apiProperties.json:11]: @sear-lin-avepoint secret look alike values are not allowed. Please replace with a dummy value.

File: certified-connectors/AvePoint Cloud Governance/apiProperties.json [hunk 2]
```diff
@@ -28,6 +58,87 @@
...
+            "description": "Sign in with Azure Active Directory"
+          },
+          "parameters": {
+            "token": {
+              "type": "oauthSetting",
+              "uiDefinition": {
+                "displayName": "Username"
+              },
+              "oAuthSettings": {
+                "clientId": "1892da54-7f57-4b6a-b6b0-fdc7f44e04c9",
```
Comment (Amjed-Ayoub) [inline certified-connectors/AvePoint Cloud Governance/apiProperties.json:80]: @sear-lin-avepoint secret look alike values are not allowed. Please replace with a dummy value.

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]

File: certified-connectors/AvePoint Cloud Governance/apiProperties.json (hunk 1 above)
Comment (sear-lin-avepoint) [reply certified-connectors/AvePoint Cloud Governance/apiProperties.json:11]: thanks for you reminder,  already replaced.

File: certified-connectors/AvePoint Cloud Governance/apiProperties.json (hunk 2 above)
Comment (sear-lin-avepoint) [reply certified-connectors/AvePoint Cloud Governance/apiProperties.json]: thanks for you reminder,  already replaced.

File: certified-connectors/AvePoint Cloud Governance/apiProperties.json (hunk 1 above)
Comment (Amjed-Ayoub) [inline certified-connectors/AvePoint Cloud Governance/apiProperties.json:11]: @sear-lin-avepoint is it possible to replace this clientId with a dummy data?

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]

File: certified-connectors/AvePoint Cloud Governance/apiProperties.json (hunk 1 above)
Comment (sear-lin-avepoint) [reply certified-connectors/AvePoint Cloud Governance/apiProperties.json]: @Amjed-Ayoub  sorry,  i though it was replaced,  just pushed a commit.

**Review by Amjed-Ayoub — APPROVED** [review]
Hello @sear-lin-avepoint,
I hope you are doing well.
Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) or [update your submission](https://learn.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#update-submission-prior-to-certification) using your latest commit ID, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio. Please [make sure that the icon meets all the requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file.
After making the submission in ISV Studio, please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6.
Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector. 
We expect all tests to be completed within 1-2 weeks.
If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions.
Please let me know if you have any questions or run into any problems while creating or updating your submission.
Thank you very much for working with us.

**Pull request merged**